tests/test_doublyrobust.py

```diff
@@ -315,6 +335,266 @@ def test_sklearn_in_tmle_missing(self, mf):
         npt.assert_allclose(tmle.odds_ratio_ci, [0.213980, 0.978331], rtol=1e-4)
 
 
+class TestStochasticTMLE:
+
+    @pytest.fixture
+    def df(self):
+        df = ze.load_sample_data(False)
+        df[['cd4_rs1', 'cd4_rs2']] = ze.spline(df, 'cd40', n_knots=3, term=2, restricted=True)
+        df[['age_rs1', 'age_rs2']] = ze.spline(df, 'age0', n_knots=3, term=2, restricted=True)
+        return df.drop(columns=['cd4_wk45']).dropna()
+
+    @pytest.fixture
+    def cf(self):
+        df = ze.load_sample_data(False)
+        df[['cd4_rs1', 'cd4_rs2']] = ze.spline(df, 'cd40', n_knots=3, term=2, restricted=True)
+        df[['age_rs1', 'age_rs2']] = ze.spline(df, 'age0', n_knots=3, term=2, restricted=True)
+        return df.drop(columns=['dead']).dropna()
+
+    @pytest.fixture
+    def simple_df(self):
+        expected = pd.DataFrame([[1, 1, 1, 1, 1],
+                                 [0, 0, 0, -1, 2],
+                                 [0, 1, 0, 5, 1],
+                                 [0, 0, 1, 0, 0],
+                                 [1, 0, 0, 0, 1],
+                                 [1, 0, 1, 0, 0],
+                                 [0, 1, 0, 10, 1],
+                                 [0, 0, 0, -5, 0],
+                                 [1, 1, 0, -5, 2]],
+                                columns=["W", "A", "Y", "C", "S"],
+                                index=[1, 2, 3, 4, 5, 6, 7, 8, 9])
+        return expected
+
+    def test_error_continuous_exp(self, df):
+        with pytest.raises(ValueError):
```

Comment:
tests like this have bitten me before: the call being tested could fail for _many_ reasons and throw a `ValueError`. You probably are hoping it fails for a specific `ValueError` - I suggest using the `match` parameter in `pytest.raises` to narrow it down.